src/mame/drivers/hp80.cpp

```diff
@@ -243,6 +242,20 @@ void hp85_state::machine_reset()
 	m_timer_idx = 0;
 	m_clk_busy = false;
 	update_irl();
+
+	// Load optional ROMs (if any)
+	// All entries in rombank (01-FF) initially not present
+	m_rombank->configure_entries(1 , 255 , m_empty_bank , 0);
```

Comment:
This doesn't really look like best practice to me.  It would be better to supply an address space to the slot devices, and allow the cards to install handlers into the address space.  Use the stuff in `bankdev.h` to access the window into the address space.  See e.g. `src/devices/bus/isa` or `src/devices/bus/intellec4` for examples of cards that install themselves into address spaces provided by the host.